util/perl/OpenSSL/config.pm

```diff
@@ -202,8 +202,8 @@ sub is_sco_uname {
     }
     close UNAME;
     return "" if $line eq '';
-    my @fields = split($line);
-    return $fields[2];
+    my @fields = split('\s+', $line);
```

Comment:
`/\s+/` would be better, as that makes it more obviously a pattern...
(had this line been repeated more than once, there are also speed impacts, since a properly marked regexp gets compiled once at perl compile file, while a string gets converted to a compiled regexp every time it's used, i.e. in run-time)